src/cmdstan/command.hpp

```diff
@@ -702,14 +691,14 @@ int command(int argc, const char *argv[]) {
 
     std::vector<std::string> p_names;
     model.constrained_param_names(p_names, false, false);
-    for (size_t i = 1; i < p_names.size(); i++) {
+    for (size_t i = 1; i < p_names.size(); ++i) {
       output_stream << "g_" << p_names[i] << ",";
     }
     output_stream << "g_" << p_names.back() << "\n";
     try {
       double lp;
       std::vector<double> gradients;
-      for (size_t i = 0; i < num_par_sets; i++) {
+      for (size_t i = 0; i < num_par_sets; ++i) {
```

Comment:
Can't this just iterate over the elements of `params_r_ind` or is it not the same size as `num_par_sets`?  If not, then this could use some doc explaining the indexing.